test/functional/pdo_sqlsrv/MsCommon.inc

```diff
@@ -8,30 +8,6 @@
 
```

Comment:
Where is the code below going? Is it already in MsCommon_mid-refactor.inc?

Reply:
yes, these were old and not used anymore